notebooks/Atlas & Kai for Mongo Side-by-Side/meta.toml

```diff
@@ -0,0 +1,3 @@
+[meta]
```

Comment:
My preference would be to use json for these config files

Reply:
There are two reasons I picked `toml` for this: 1) I wanted these files to be easy for humans to read and write, and 2) Python package installation and most Python tooling is moving to `toml` as their configuration file format. I don't feel like JSON fits the first condition. I don't know that it really matters much from the repo side though. I think we should version notebook releases using Github actions and the actions can build whatever assets we need in whatever format is best for the tooling on the consumer side.

Reply:
Alright fair enough